packages/web/utils/helper.js

```diff
@@ -327,3 +327,26 @@ export const stringToLocaleDate = (date, options = {}) =>
  */
 export const stringToLocaleTime = (date, options = {}) =>
 	new Date(date).toLocaleTimeString('pt-br', { ...options });
+
+/**
+ * Handle phone number mask, necessary to alternate between masks when phone has 9 digits
```

Comment:
Is this the true description? I'm confused due it's the same of `beforeMaskedValueChange` but seems to have another purpose 😅